src/OSPSuite.Core/Comparison/ExpressionProfileDiffBuilder.cs

```diff
@@ -0,0 +1,19 @@
+﻿using OSPSuite.Core.Domain.Builder;
+
+namespace OSPSuite.Core.Comparison
+{
+   public class ExpressionProfileBuildingBlockDiffBuilder : StartValueBuildingBlockDiffBuilder<ExpressionParameter>
+   {
+      public ExpressionProfileBuildingBlockDiffBuilder(ObjectBaseDiffBuilder objectBaseDiffBuilder, EnumerableComparer enumerableComparer) : base(objectBaseDiffBuilder, enumerableComparer)
```

Comment:
I think some properties are missing.
Type, PKSImVersion etc

Reply:
In short, evey single propoerty should be compared, based on the settings of the comparison

Reply:
fixed

Reply:
so the settings of the comparison are not used./
See ParameterDiffBuilder